docs/source/plugin_dev/cli.rst

```diff
@@ -136,3 +136,173 @@ name
 
     If file extension is included in the name and the file extension is a valid `packed plugin <plugin_format.html#packed-plugin>`__ extension (``.mcdr`` or ``.pyz``), then the included file extension will be used. Otherwise the default ``.mcdr`` file extension will be appended to the end
 
+init_plugin
+^^^^^^^^^^^
+
+.. code-block::
+
+    python -m mcdreforged [-q] init_plugin [-h] [-p PATH] [-i ID] [-n NAME] [-d DESCRIPTION] [-a AUTHOR] [-l LINK] [-r RESOURCES] [-e ENTRYPOINT] [-A ARCHIVE_NAME]
+
+Prepare the workspace of MCDR plugin
+
+See :doc:`metadata <metadata>` for more information.
+
+quiet
+"""""
+
+    ``-q``, ``--quiet``
+
+    If use quiet mode, then will disable any ask and output, it will use the argument value or the default value
+
+path
+""""
+
+    ``-p PATH``, ``--path PATH``
+
+    The work directory which the plugin will be in
+
+    default: current directory
+
+id
+""
+
+    ``-i ID``, ``--id ID``
+
+    The identity string of your plugin. It should consist of lowercase letters, numbers and underscores with a length of 1 to 64
+
+    default: current directory name (may not vaild)
+
+name
+""""
+
+    ``-n NAME``, ``--name NAME``
+
+    The name of your plugin.
+
+    default: plugin id
+
+description
+"""""""""""
+
+    ``-d DESCRIPTION``, ``--description DESCRIPTION``
+
+    The description of you plugin.
+
+    default: This is a plugin for MCDR
+
+author
+""""""
+
+    ``-a AUTHOR``, ``--author AUTHOR``
+
+    The author list of the plugins. split with ``:``
+
+    For example:
+
+    ``-a Fallen_Breath`` => ``"author": ["Fallen_Breath"]``
+
+    ``-a Fallen_Breath:Author2`` => ``"author": ["Fallen_Breath", "Author2"]``
+
+    default: None
+
+link
+""""
+
+    ``-l LINK``, ``--link LINK``
+
+    The url to your plugin.
+
+    default: None
+
+resources
+"""""""""
+
+    ``-r RESOURCES``, ``--resources RESOURCES``
+
+    The resource list, split with ``:``
+
+    For example:
+
+    ``-r lang`` => ``"resources": ["lang"]``
+
+    ``-r lang:example`` => ``"resources": ["lang", "example"]``
+
+    default: None
+
+entrypoint
+""""""""""
+
+    ``-e ENTRYPOINT``, ``--entrypoint ENTRYPOINT``
+
+    The entrypoint module of your plugin
+
+    default: None
+
+archive_name
+""""""""""""
+
+    ``-A ARCHIVE_NAME``, ``--archive_name ARCHIVE_NAME``
+
+    The file name of generated ``.mcdr`` packed plugin in command ``pack``
+
+    default: None
+
+Create commonly used folders and generate default configure files, including:
+
+* <plugin_id>/<entrypoint>.py
+* mcdreforged.plugin.json
+* requirements.txt
+
+Example:
+""""""""
+
+.. code-block::
+
+    $ mkdir hello_world
+    $ cd hello_world
+    $ python3 -m mcdreforged init_plugin
+    Plugin work directory (default "."): 
```

Comment:
"Plugin work directory" is a bit ambiguous for users since it's not really a "work directory"

Reply:
workspace will be a better choice, just like what you write in the document of the `path` param